### PR TITLE
Update ThePirateBaySe.py added 3D ,bluray , and heb dubbed movies.

### PR DESCRIPTION
**Pull request by yokod**

### CODE REVIEW

File: resources/contenters/ThePirateBaySe.py [hunk 1]
```diff
@@ -53,9 +66,6 @@ class ThePirateBaySe(Content.Content):
     def isTracker(self):
         return True
 
-    def isSearcher(self):
```
Comment (DiMartinoXBMC) [inline resources/contenters/ThePirateBaySe.py]: why did you delete that?

Comment (yokod) [inline resources/contenters/ThePirateBaySe.py]: I didn't mean to. Sorry
